src/common/Browser.ts

```diff
@@ -319,7 +319,7 @@ export class Browser extends EventEmitter {
   async createIncognitoBrowserContext(
     options: BrowserContextOptions = {}
   ): Promise<BrowserContext> {
-    const { proxyServer = '', proxyBypassList = [] } = options;
+    const { proxyServer, proxyBypassList } = options;
```

Comment:
Omitting the defaults here causes the browser-level settings to be respected if these aren't provided.